src/scriptworker/cot/verify.py

```diff
@@ -1037,8 +1039,16 @@ async def _get_additional_hg_action_jsone_context(parent_link, decision_link):
 async def _get_additional_hgpush_info(decision_link):
     pushlog_info = await get_pushlog_info(decision_link)
     pushlog_id = list(pushlog_info["pushes"].keys())[0]
-    push_comment = pushlog_info["pushes"][pushlog_id]["changesets"][0]["desc"]
-    return {"id": pushlog_id, "comment": push_comment, "date": pushlog_info["pushes"][pushlog_id]["date"], "user": pushlog_info["pushes"][pushlog_id]["user"]}
+    first_revision = pushlog_info["pushes"][pushlog_id]["changesets"][0]
+    push_comment = first_revision["desc"]
```

Comment:
Shouldn't this be the tip commit's description, to preserve the previous behavior?

Reply:
Oh yeah, definitely! Thanks for pointing that out. Fixed!